src/components/submission/metadata-row.vue

```diff
@@ -159,7 +159,11 @@ export default {
     max-width: 250px;
   }
 
-  .state-and-actions { min-width: 205px; }
+  .state-and-actions {
+    // Ensure that the column is wide enough that the .btn-group does not wrap.
+    min-width: 205px;
+    &:lang(id) { min-width: 221px; }
```

Comment:
On my machine, 208px was sufficient to prevent wrapping. However, I wanted to add a little wiggle room in case the users' fonts are a little different from mine. I also wanted the user to be able to see at least a fraction of the column content outside the buttons. In the screenshot, you can see that part of the review state icon is still visible to the left of the buttons. I didn't want to make the column overly wide though, so it was a balancing act.